src/ResourceManager/ServiceFabric/Commands.ServiceFabric/packages.config

```diff
@@ -2,9 +2,9 @@
 <packages>
   <package id="Microsoft.Azure.KeyVault" version="2.3.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.KeyVault.WebKey" version="2.0.6" targetFramework="net452" />
-  <package id="Microsoft.Azure.Management.Compute" version="14.0.0-prerelease" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Compute" version="17.3.0" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.KeyVault" version="2.3.0-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.Management.Network" version="8.1.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Network" version="15.1.0-preview" targetFramework="net452" />
```

Comment:
@juhacket something that doesn't need to be addressed now, but we should look at in an upcoming sprint: we need to remove ServiceFabric's dependency on other services' management libraries. We can do this by (1) evaluating whether or not ServiceFabric needs the management library and (2) generating pieces of each library that ServiceFabric (and potentially other services) consume in their module and put that in `src/Common` along with other generated libraries.

Reply:
@juhacket here's the corresponding issue to track this work: https://github.com/Azure/azure-powershell/issues/5365